web/packages/teleport/src/Roles/RoleEditor/RoleEditorAdapter.tsx

```diff
@@ -106,55 +108,59 @@ export function RoleEditorAdapter({
         )}
       </Flex>
       <Flex flex="1" alignItems="center" justifyContent="center" m={3}>
-        {/* Same width as promo image + border */}
-        <Box maxWidth={promoImageWidth + 2 * 2} minWidth={300}>
-          <H1 mb={2}>Coming soon: Teleport Policy saves you from mistakes</H1>
-          <Flex mb={4} gap={4} flexWrap="wrap" justifyContent="space-between">
-            <Box flex="1" minWidth="30ch">
-              <P>
-                Teleport Policy will visualize resource access paths as you
-                create and edit roles so you can always see what you are
-                granting before you push a role into production.
-              </P>
-            </Box>
-            <Flex flex="0 0 auto" alignItems="start">
-              {!cfg.isPolicyEnabled && (
-                <>
-                  <ButtonLockedFeature noIcon py={0} width={undefined}>
-                    Contact Sales
-                  </ButtonLockedFeature>
-                  <ButtonSecondary
-                    as="a"
-                    href="https://goteleport.com/platform/policy/"
-                    target="_blank"
-                    ml={2}
-                  >
-                    Learn More
-                  </ButtonSecondary>
-                </>
-              )}
+        {/* TODO (avatus) this component will not be rendered until the Access Diff feature is implemented */}
+        {roleDiffProps ? (
+          <roleDiffProps.RoleDiffComponent />
+        ) : (
+          <Box maxWidth={promoImageWidth + 2 * 2} minWidth={300}>
```

Comment:
the diff is a bit wild on this one due to a lot of similar child components, but i didnt change anything with the structure of the Coming Soon box, just moved it into the tertiary

Reply:
Can you make this part a separate component that is passed in as `RoleDiffComponent` for the OSS version?

Reply:
I can make it a separate component, sure. I didn't make it OSS only since `e` clusters will still need this if TAG isn't enabled

Reply:
Ah fair enough. I think putting it in a separate component and importing it from `e` makes it a bit more obvious what's happening on both the e and OSS side

Reply:
Makes sense. Ill update it